src/main/java/hometask3/shape/Rectangular.java

```diff
@@ -0,0 +1,17 @@
+package hometask3.shape;
+
+public abstract class Rectangular implements Shape{
+    int a;
+    int b;
+
+    public Rectangular(int a) {
```

Comment:
Please remove this constructor, Rectangular should have only one with width and height and will be overriden by child-classes that need another "simplified" constructor